main.go

```diff
@@ -86,13 +87,6 @@ func main() {
 
 	ctrl.SetLogger(zapr.NewLogger(ctrlZap))
 
-	klogZap, err := zaputil.BuildForKlog(zflags.GetConfig(), flag.CommandLine)
-	if err != nil {
-		log.Error(err, "Failed to build zap logger for klog, those logs will not go through zap")
-	} else {
-		klog.SetLogger(zapr.NewLogger(klogZap).WithName("klog"))
-	}
-
```

Comment:
Is there a reason to remove the `klog` formatting? Logs from controller-runtime and other libraries should still be coming through that